public/app/features/admin/state/reducers.test.ts

```diff
@@ -70,7 +70,7 @@ describe('LDAP page reducer', () => {
                 available: true,
                 host: 'localhost',
                 port: 389,
-                error: null,
+                error: (null as unknown) as string,
```

Comment:
What is happening here? :D

Reply:
yeah, so ugly but the recommended way according to TS :)

Reply:
```console
If this was intentional, convert the expression  to 'unknown' first.
```

Reply:
Ok, makes more sense now, thank you!